addons/l10n_it_edi/models/res_partner.py

```diff
@@ -35,9 +35,7 @@ class ResPartner(models.Model):
 
     def _l10n_it_edi_is_public_administration(self):
         """ Returns True if the destination of the FatturaPA belongs to the Public Administration. """
-        self.ensure_one()
```

Comment:
should the ensure_one not remain? self.country_id.code will expect singleton
https://github.com/odoo/odoo/pull/142596/files#diff-7a314f6fb9514fd24adb9e88e00fe802ff75743c7227d648a65f44ca34de9883R1100 seems to indicate it may not be a singleton